test/framework/container_engine.go

```diff
@@ -103,7 +103,7 @@ func (containerEngine *containerEngineStruct) PushImage(imageTag string) Contain
 
 func (containerEngine *containerEngineStruct) BuildImage(projectLocation, imageTag string) ContainerEngine {
 	if containerEngine.err == nil {
-		_, containerEngine.err = CreateCommand(containerEngine.engine, containerEngine.buildCommand, "--tag", imageTag, ".").InDirectory(projectLocation).WithLoggerContext(containerEngine.namespace).Execute()
+		_, containerEngine.err = CreateCommand(containerEngine.engine, containerEngine.buildCommand, "--tag", imageTag, ".").InDirectory(projectLocation).WithLoggerContext(containerEngine.namespace).WithRetry(NumberOfRetries(3)).Execute()
```

Comment:
Shouldn't we set also RetryDelay ?

Reply:
IMHO it is not needed in this case. In case of some temporary issue I think that we can retry immediately.
RetryDelay was introduced as an additional functionality for command, not needed in this case.

Reply:
ok so if no value then the sleep is not happening right ?

Reply:
Yes